app/app.rb

```diff
@@ -4,6 +4,7 @@
 require "sinatra/streaming"
 require 'sinatra/pundit'
 
+require "active_support"
```

Comment:
Is there any way to not import monkeypatches from activesupport? So we don't depend on it much by accident. Maybe we can seal the import in a module?